portlet-api/pom.xml

```diff
@@ -103,6 +103,21 @@
                   </bottom>
                </configuration>
             </plugin>
+            <plugin>
+               <groupId>org.apache.felix</groupId>
+               <artifactId>maven-bundle-plugin</artifactId>
+               <version>3.2.0</version>
+               <extensions>true</extensions>
+               <configuration>
+                  <instructions>
+                     <Import-Package>
+                        javax.enterprise.context;version="[1.1,2)";resolution:=optional,
+                        javax.inject;resolution:=optional,*
+                     </Import-Package>
+                     <Provide-Capability>osgi.contract; osgi.contract=JavaPortlet; version:List&lt;Version&gt;="3,2,1"; uses:="javax.enterprise.context,javax.inject,javax.servlet,javax.servlet.http,javax.xml.namespace,org.w3c.dom"</Provide-Capability>
```

Comment:
To be clear, don't include the packages used by the API... only include the packages provided (exported) by the API.